libopenfpga/libfpgaaesencryption/test/AES_main.cpp

```diff
@@ -0,0 +1,211 @@
+#include "AESfunctions.h"
+#include <fstream> 
+#include <string>
+#include <sstream>
+#include <algorithm>
+#include<cctype>
+
+//String conversion function
+string convertToString(char* a, int size)
+{
+    int i;
+    string s = "";
+    for (i = 0; i < size; i++) {
+        s = s + a[i];
+    }
+    return s;
+}
+
+//Function to convert binary to Decimal
+
+int binaryToDecimal(int n)
```

Comment:
We already have the function in the openfpga utils library: 

https://github.com/lnis-uofu/OpenFPGA/blob/master/libopenfpga/libopenfpgautil/src/openfpga_decode.h

If not, we should add the function to the util library, and just call it here.